lib/matplotlib/path.py

```diff
@@ -667,14 +667,35 @@ def intersects_bbox(self, bbox, filled=True):
 
     def interpolated(self, steps):
         """
-        Return a new path resampled to length N x *steps*.
+        Return a new path with each segment divided into *steps* parts.
 
-        Codes other than `LINETO` are not handled correctly.
+        Codes other than `LINETO` `MOVETO` and `CLOSEPOLY` are not handled correctly.
```

Comment:
```suggestion
        Codes other than `LINETO`, `MOVETO`, and `CLOSEPOLY` are not handled correctly.
```
(at least the first, but since I like Oxford-commas...)